### PR TITLE
dmaker.airfresh.t2017: Fix display orientation control

### PR DESCRIPTION
**Pull request by skvalex**
- Set display orientation is now working
- Updated README to correct mistypes

### CODE REVIEW

File: custom_components/xiaomi_miio_airpurifier/fan.py [hunk 1]
```diff
@@ -2013,15 +2014,15 @@ async def async_set_display_off(self):
...
+    async def async_set_display_orientation(self, display_orientation: str):
         """Set the display orientation."""
         if self._device_features & FEATURE_SET_DISPLAY_ORIENTATION == 0:
             return
 
         await self._try_command(
             "Setting the display orientation of the miio device failed.",
             self._device.set_display_orientation,
-            AirfreshT2017DisplayOrientation[orientation.title()],
+            AirfreshT2017DisplayOrientation[display_orientation],
```
Comment (syssi) [inline custom_components/xiaomi_miio_airpurifier/fan.py:2025]: Could you use `display_orientation.title()` here?

Comment (skvalex) [reply custom_components/xiaomi_miio_airpurifier/fan.py:2025]: No, it makes `LandscapeLeft` to `Landscapeleft`, which doesn't work

Comment (syssi) [reply custom_components/xiaomi_miio_airpurifier/fan.py:2025]: Ups!

**Comment by syssi** [comment]
Thanks for your support! Good catch!

**Pull request merged**